docs/use-cases/collect-sensor-data.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Collect and view sensor data from any machines"
+linkTitle: "Collect sensor data"
+weight: 29
+type: "docs"
+images: ["/services/icons/data-query.svg"]
+description: "Gather sensor data, sync it to the cloud, and query it from the Viam app."
+modulescript: true
+# SME: Devin Hilly
+---
+
+You can use the data management service to capture sensor or time-series data from any machine and sync that data to the cloud.
```

Comment:
```suggestion
You can use the data management service to capture sensor data from any machine and sync that data to the cloud.
```
"sensor data" and "time series data" are not mutually exclusive right? I'm thinking time-series is an unfamiliar technical term to some folks (including myself), so maybe we don't need it in the first sentence of this page.

Also [nit] most of the internet seems not to hyphenate time series.